papers/Phi Fei Fo Fum (WIP)/paper.Rmd

```diff
@@ -0,0 +1,286 @@
+---
+title: "Phi, Fei, Fo, Fum: Effect Sizes for Chi-squared Tests"
+
+authors:
+- affiliation: 1
+  name: Mattan S. Ben-Shachar
+  orcid: 0000-0002-4287-4801
+- affiliation: 2
+  name: Indrajeet Patil
+  orcid: 0000-0003-1995-6531
+- affiliation: 3
+  name: Rémi Thériault
+  orcid: 0000-0003-4315-6788
+- affiliation: 4
+  name: Brenton M. Wiernik
+  orcid: to add
+- affiliation: 5
+  name: Daniel Lüdecke 
+  orcid: to add
+
+affiliations:
+- index: 1
+  name: XXX
+- index: 2
+  name: Center for Humans and Machines, Max Planck Institute for Human Development, Berlin, Germany  
+- index: 3
+  name: Université du Québec à Montréal, Montréal, Québec
+- index: 4
+  name: Independent researcher
+- index: 5
+  name: to add
+
+output: 
+  pdf_document:
+    keep_tex: true
+    latex_engine: xelatex
+mainfont: Times New Roman
+
+bibliography: paper.bib
+csl: apa.csl
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+
+library(effectsize)
+# options(es.use_symbols = TRUE)
+```
+
+# Abstract
+
+In both theoretical and applied research, it is often of interest to assess the strength of an observed association. Existing guidelines also frequently recommend going beyond null-hypothesis significance testing and to report effect sizes and their confidence intervals. As such, measures of effect sizes are increasingly reported, valued, and understood. Beyond their value in shaping the interpretation of the results from a given study, reporting effect sizes is critical for meta-analyses, which rely on their aggregation. We here review the most common effect sizes for analyses of categorical variables that use the $\chi^2$ (chi-square) statistic, and introduce a new effect size---`r "\U05E4"` (Fei, pronounced /fej/ or "fay"). We demonstrate the implementation of these measures and their confidence intervals via the `{effectsize}` package [@benshachar2020effectsize] in the R programming language.
+
+# Introduction
+
+Over the last two decades, there has been growing concerns about the so-called replication crisis in psychology and other fields [@OSC2015estimating; @camerer2018evaluating]. As a result, the scientific community has paid increasing attention to the issue of replicability in science, as well as to to good research and statistical practices.
+
+In this context, many have highlighted the limitations of null-hypothesis significance testing and called for more modern approaches to statistics [@cumming2014new]. One such recommendation coming for example from the New Statistics movement is to report effect sizes and their corresponding confidence intervals, and to increasingly rely on meta-analyses to increase confidence in those estimations. These recommendations are meant to complement (or even replace, according to some) null-hypothesis significance testing and would help transition toward a "cumulative quantitative discipline".
```

Comment:
I think @rempsyc added this?

Reply:
I don't think it's appropriate to define the New Statistics here since we already give a reference to it [@cumming2014new; _The New Statistics: Why and How_]. It should be enough no? If you'd like you can move that reference to the next sentence. And maybe it's not an actual movement it's not that clear so we could remove that word if we want.

Reply:
> If you'd like you can move that reference to the next sentence

Yeah, maybe that's clearer

Reply:
Put "new statistics" in quotes as it's a sort of brand name by Cumming and co to refer to estimation + interval approach